metricbeat/scripts/docs_collector.py

```diff
@@ -83,6 +83,10 @@ def collect(beat_name):
 
             module_file += "----\n\n"
 
+        # HTTP helper
```

Comment:
@exekias Awesome. Was thinking of exactly that when I saw the notification for the PR.